### ISSUE TITLE
Compiling on OS X Yosemite

### ISSUE THREAD

**Issue by informer2000** [tatsuhiro-t/nghttp2#163]
Finally got nghttp2 to compile on Yosemite! One of errors I was getting was related to asio-sv2.cc.

```
asio-sv2.cc:97:53: error: no member named 'st_mtim' in 'stat'
            header{"last-modified", http_date(stbuf.st_mtim.tv_sec)});
```

Had to change it to

```
header{"last-modified", http_date(stbuf.st_mtime)});
```

The other two issues were related to openssl and libxml2. Both were installed using homebrew. For some reason the symlinks `/usr/bin/openssl` and `/usr/lib/libssl.dylib` were not updated to point to the newly installed version and I was getting an error because of the old header files.

The libxml2 issue was related to the am_path_xml2.m4 file. I managed to fix it by downloading the file from http://keithr.org/eric/src/whoisserver-nightly/m4/am_path_xml2.m4 and placing it in the `m4` subdirectory, following the instructions in http://blog.kazuhooku.com/2014/11/memo-installing-nghttp2-onto-osx.html

**Comment by tatsuhiro-t** [comment]
Thanks for reporting this.  The fix you proposed was committed via 3144f7d.
libxml2.m4 issue is repeatedly reported.  I'm now considering to include it to git repo.

**Comment by tatsuhiro-t** [comment]
libxml2.m4 is now available in m4 directory with commit c506386
